core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1387,6 +1387,27 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
   }
 
   /**
+    * Return a list of file paths that are added to resources.
+    * If file paths are provided, return the ones that are added to resources.
+    */
+  def listFiles(files: Seq[String] = Seq.empty[String]): Seq[String] = {
```

Comment:
i think this one should not take any parameter, and if you need filtering, just do it in ListFilesCommand